prime-router/src/main/kotlin/cli/tests/BasicTests.kt

```diff
@@ -655,6 +655,7 @@ class QualityFilter : CoolTest() {
         // ALLOW ALL
         ugly("\nTest the allowAll QualityFilter")
         val fakeItemCount = 5
+        var expectItemCount = fakeItemCount
```

Comment:
This one confused me when I saw the same value.  I suggest you move this assignment down to where you need it like the others, so it makes more sense why you are using it.

Reply:
Done.